libs/output-mapping/src/Writer/Helper/ConfigurationMerger.php

```diff
@@ -100,4 +104,20 @@ private static function mergeKeyValueArray(array $manifest, array $mapping): arr
         }
         return $manifest;
     }
+
+    private static function mergePrimaryKeyAndSchema(array $manifestSchema, array $primaryKeys): array
+    {
+        foreach ($primaryKeys as $primaryKey) {
+            $manifestColumn = array_filter($manifestSchema, fn($v) => $v['name'] === $primaryKey);
+            if (!$manifestColumn) {
+                continue;
+            }
```

Comment:
Akorát, až bude mít někdo v konfiguraci zas už zwebalizovanou podobu, tak budeme v loji a PK to odebere.

Nemělo by to místo continue spíš skončit chybou?

Reply:
tak webalizujeme všechno my tak by to nmuselo dělat problémy

Reply:
no je otázkou no jestli skončit chybou nebo takhle

Reply:
Jo už to vidím. My to nejprve webalizujeme a pak až mergueme

https://github.com/keboola/platform-libraries/blob/54d28c97f60c03a58a47f4710404c1f4e16b54dc/libs/output-mapping/src/TableLoader.php#L90-L101

Takže z tohodle pohledu by to mělo být ok

Reply:
Místo continue by to ale fakt mělo vyhazovat vyjímku.

Reply:
Protože by mohla nastat situace

- mám v konfiguraci OM jako PK sloupec "test"
- manifest komponenty z nějakého důvodu negeneruje schema pro všechny sloupce

V tomhle případe by tato implementace "test" sloupec potichu vynechala a z tabulky by zmizel PK.

Reply:
tak to vyhazuuje Exceptionu